io/disk/disktest.py

```diff
@@ -64,37 +64,38 @@ def setUp(self):
         """
         smm = SoftwareManager()
         if not smm.check_installed("gcc") and not smm.install("gcc"):
-            self.cancel('Gcc is needed for the test to be run')
+            self.cancel("Gcc is needed for the test to be run")
         # Log of all the disktest processes
-        self.disk_log = os.path.abspath(os.path.join(self.outputdir,
-                                                     "log.txt"))
+        self.disk_log = os.path.abspath(os.path.join(self.outputdir, "log.txt"))
         self.fs_create = False
-        self.raid_needed = self.params.get('raid', default=False)
+        self.raid_needed = self.params.get("raid", default=False)
         self.raid_create = False
-        self.disk = self.params.get('disk', default=None)
-        self.dir = self.params.get('dir', default=None)
-        self.fstype = self.params.get('fs', default='ext4')
-        self.raid_name = '/dev/md/sraid'
+        self.disk = self.params.get("disk", default=None)
+        self.dir = self.params.get("dir", default="/mnt")
+        self.fstype = self.params.get("fs", default="ext4")
+        self.raid_name = "/dev/md/sraid"
         self.err_mesg = []
 
-        if self.fstype == 'btrfs':
+        if self.fstype == "btrfs":
             ver = int(distro.detect().version)
             rel = int(distro.detect().release)
-            if distro.detect().name == 'rhel':
+            if distro.detect().name == "rhel":
                 if (ver == 7 and rel >= 4) or ver > 7:
-                    self.cancel("btrfs is not supported with \
-                                RHEL 7.4 onwards")
+                    self.cancel(
+                        "btrfs is not supported with \
+                                RHEL 7.4 onwards"
+                    )
         if self.raid_needed:
-            if not smm.check_installed("mdadm") \
-               and not smm.install("mdadm"):
-                self.cancel('mdadm is needed for the test to be run')
+            if not smm.check_installed("mdadm") and not smm.install("mdadm"):
+                self.cancel("mdadm is needed for the test to be run")
 
-        self.vgname = 'avocado_vg'
-        self.lvname = 'avocado_lv'
+        self.vgname = "avocado_vg"
+        self.lvname = "avocado_lv"
```

Comment:
@AYUSHJAIN951 is there any specific case where this styling is a must required ?
if not then keeping existing style as it is better than getting into a mesh.

Reply:
These changes were recommended by "Python Black".
But as its not so much needed i have removed these changes and pushed again.

Reply:
@AYUSHJAIN951 you have still has the changes here, please check if it is still a mandatory change.

Reply:
@Naresh-ibm  Can you please check again as it's not there now.